src/index.js

```diff
@@ -0,0 +1,347 @@
+    const lang = localStorage.getItem("lang");
+    const city = localStorage.getItem("city");
+    showSearchCity(city, lang);
+    if (isFarengeit) {
+        buttonFarenheit.classList.add('active');
+        buttonCelsius.classList.remove('active');
+    } else {
+        buttonFarenheit.classList.remove('active');
+        buttonCelsius.classList.add('active');
+    }
+};
+
+function getAdress(latitudeNow, longitudeNow) {
+    return fetch(`https://api.opencagedata.com/geocode/v1/json?&q=${latitudeNow}+${longitudeNow}&key=7ec9383669c44f36be73334edd48f8b1`)
+        .then((response) => response.json());
+};
+
+async function showAdress(latitudeNow, longitudeNow) {
+    try {
+        const adress = await getAdress(latitudeNow, longitudeNow);
+        const locations = adress.results[0].components;
+        const city = locations.city;
+        const { country } = locations;
+        locationCity.textContent = `${city}, ${country}`;
+        const lang = localStorage.getItem('lang')
+        showWeatherNow(city, lang);
+    } catch (error) {
+        console.log(error);
+    }
+};
+
+function searchCity(city, lang) {
+    return fetch(`https://api.opencagedata.com/geocode/v1/json?q=${city}&language=${lang}&key=7ec9383669c44f36be73334edd48f8b1`)
+        .then((response) => response.json());
+};
+
+async function showSearchCity(city, lang) {
+    try {
+        if (!city) {
+            city = inputCity.value;
+        }
+        const adress = await searchCity(city, lang);
+
+        if (adress) {
+
+            const result = adress.results[0].components;
+            const city = result.city ? result.city : result.town ? result.town : result.village;
+
+            const { country } = result;
+            locationCity.textContent = `${city}, ${country}`;
+
+            const zyk = adress.results[0].geometry;
+            const LatitudeNow = zyk.lat.toFixed(2); //show lat and lng formats a number using fixed-point notation 
+            const LongitudeNow = zyk.lng.toFixed(2);
+
+            getCoordinats(LatitudeNow, LongitudeNow);
+            showWeatherNow(city, lang);
+            getMap(LatitudeNow, LongitudeNow);
+
+        }
+    } catch (error) {
+        console.log(error);
+    }
+};
+
+const getWeatherLatLong = async (latitudeNow, longitudeNow) =>
+    fetch(`https://api.openweathermap.org/data/2.5/forecast?lat=${latitudeNow}&lon=${longitudeNow}&units=metric&appid=c3ee163c21d694ddab64849983b70180`)
+        .then((response) => response.json());
+
+async function showWeatherLatLong(LatitudeNow, LongitudeNow) {
+    try {
+        const weather = await getWeatherLatLong(LatitudeNow, LongitudeNow);
+        const city = `${weather.city.name}`;
+        showSearchCity(city);
+    } catch (error) {
+        console.log(error);
+    }
+};
+
+const getWeatherNow = async (city, lang) =>
+    fetch(`https://api.openweathermap.org/data/2.5/forecast?q=${city}&lang=${lang}&units=metric&appid=c3ee163c21d694ddab64849983b70180`)
```

Comment:
For temperature in Fahrenheit use units=imperial

Reply:
well done